### ISSUE TITLE
PHP Notice:  Undefined index: lloc

### ISSUE THREAD

**Issue by moffe42** [sebastianbergmann/phploc#125]
This is probably related to #112 

```
phploc 2.1.4 by Sebastian Bergmann.

PHP Notice:  Undefined index: lloc in /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/Analyser.php on line 312
PHP Stack trace:
PHP   1. {main}() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/phploc:0
PHP   2. Symfony\Component\Console\Application->run() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/phploc:31
PHP   3. SebastianBergmann\PHPLOC\CLI\Application->doRun() /Users/jach/Code/colourbox/library/vendor/symfony/console/Application.php:126
PHP   4. Symfony\Component\Console\Application->doRun() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/CLI/Application.php:98
PHP   5. Symfony\Component\Console\Application->doRunCommand() /Users/jach/Code/colourbox/library/vendor/symfony/console/Application.php:195
PHP   6. Symfony\Component\Console\Command\Command->run() /Users/jach/Code/colourbox/library/vendor/symfony/console/Application.php:878
PHP   7. SebastianBergmann\PHPLOC\CLI\Command->execute() /Users/jach/Code/colourbox/library/vendor/symfony/console/Command/Command.php:259
PHP   8. SebastianBergmann\PHPLOC\CLI\Command->executeSingle() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/CLI/Command.php:109
PHP   9. SebastianBergmann\PHPLOC\CLI\Command->count() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/CLI/Command.php:129
PHP  10. SebastianBergmann\PHPLOC\Analyser->countFiles() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/CLI/Command.php:233
PHP  11. SebastianBergmann\PHPLOC\Analyser->countFile() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/Analyser.php:167

Notice: Undefined index: lloc in /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/Analyser.php on line 312

Call Stack:
    0.0002     232808   1. {main}() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/phploc:0
    0.0183    2791696   2. Symfony\Component\Console\Application->run() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/phploc:31
    0.0211    3168952   3. SebastianBergmann\PHPLOC\CLI\Application->doRun() /Users/jach/Code/colourbox/library/vendor/symfony/console/Application.php:126
    0.0212    3170208   4. Symfony\Component\Console\Application->doRun() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/CLI/Application.php:98
    0.0214    3171112   5. Symfony\Component\Console\Application->doRunCommand() /Users/jach/Code/colourbox/library/vendor/symfony/console/Application.php:195
    0.0214    3171616   6. Symfony\Component\Console\Command\Command->run() /Users/jach/Code/colourbox/library/vendor/symfony/console/Application.php:878
    0.0219    3176768   7. SebastianBergmann\PHPLOC\CLI\Command->execute() /Users/jach/Code/colourbox/library/vendor/symfony/console/Command/Command.php:259
    0.0219    3177224   8. SebastianBergmann\PHPLOC\CLI\Command->executeSingle() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/CLI/Command.php:109
    0.0219    3177512   9. SebastianBergmann\PHPLOC\CLI\Command->count() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/CLI/Command.php:129
    0.0257    3813056  10. SebastianBergmann\PHPLOC\Analyser->countFiles() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/CLI/Command.php:233
    0.0257    3815440  11. SebastianBergmann\PHPLOC\Analyser->countFile() /Users/jach/Code/colourbox/library/vendor/phploc/phploc/src/Analyser.php:167
```

Minimal code snippet to reproduce the issue
``` 
<?php
class Test
{
    public function test()
    {
        function testFunction() {}

        usort($array, function ($a, $b) {
            return 1;
        });
    }
}
```

**Comment by acuthbert** [comment]
Also having this issue.

**Comment by sebastianbergmann** [comment]
The "nested function" `function testFunction() {}` throws off PHPLOC.

**Comment by sebastianbergmann** [comment]
Should be fixed in 3.0.0.
